docs/readmes/dp/debug_logs.md

```diff
@@ -7,7 +7,111 @@ id: debug_logs title: View Logs hide_title: true
+
+[TAR.XZ - example pcap of TR069 session flow between eNB and ACS (enodebd)](../assets/dp/dp_enb_tr069.pcap.tar.xz)
+
+Given the following log snippet (trimmed) from Baicells QRTB, the normal flow should consist of:
+
+1. Initial `Inform` which starts the TR069 session between eNB and ACS (enodebd)
+    1. `2022-04-25 15:41:17,549 DEBUG Handling TR069 message: Inform {...}`
+    1. `2022-04-25 15:41:17,550 DEBUG (Inform msg) Received parameter: Device.DeviceInfo.HardwareVersion = ...`
+    1. `[Inform msg output continues]`
+    1. `2022-04-25 15:41:17,557 DEBUG State transition from <WaitInformState> to <wait_empty>`
+    1. `2022-04-25 15:41:17,558 DEBUG Sending TR069 message: InformResponse {'MaxEnvelopes': '1'}`
+1. Sequence of `GetParameterValues` messages (Transient, Normal, Object)
+    1. `2022-04-25 15:41:22,608 DEBUG 10.0.2.243 - "POST / HTTP/1.1" 200 419`
+    1. `2022-04-25 15:41:22,612 DEBUG Handling TR069 message: DummyInput {}`
+    1. `2022-04-25 15:41:22,612 DEBUG State transition from <WaitEmptyMessageState> to <check_fw_upgrade_download>`
+    1. `12022-04-25 15:41:22,613 DEBUG Skipping FW Download for eNB [XXXXXXXXXXXXX], missing firmware upgrade config in enodebd.yml.`
+    1. `2022-04-25 15:41:22,613 DEBUG State transition from <CheckFirmwareUpgradeDownloadState> to <get_transient_params>`
+    1. `2022-04-25 15:41:22,613 DEBUG State transition from <SendGetTransientParametersState> to <wait_get_transient_params>`
+    1. `2022-04-25 15:41:22,613 DEBUG Sending TR069 message: GetParameterValues {...}`
+    1. `2022-04-25 15:41:22,617 DEBUG 10.0.2.243 - "POST / HTTP/1.1" 200 912`
+    1. `2022-04-25 15:41:22,662 DEBUG Handling TR069 message: GetParameterValuesResponse {...}`
+    1. `2022-04-25 15:41:22,662 DEBUG Fetched Transient Params: {...}`
+    1. `2022-04-25 15:41:22,663 DEBUG State transition from <WaitGetTransientParametersState> to <get_params>`
+    1. `2022-04-25 15:41:22,664 DEBUG State transition from <GetParametersState> to <wait_get_params>`
+    1. `2022-04-25 15:41:22,664 DEBUG Sending TR069 message: GetParameterValues {...}`
+    1. `2022-04-25 15:41:22,671 DEBUG 10.0.2.243 - "POST / HTTP/1.1" 200 3242`
+    1. `2022-04-25 15:41:22,718 DEBUG Handling TR069 message: GetParameterValuesResponse {...}`
+    1. `2022-04-25 15:41:22,719 DEBUG Received CPE parameter values: {...}`
+    1. `2022-04-25 15:41:22,719 DEBUG State transition from <WaitGetParametersState> to <get_obj_params>`
+    1. `2022-04-25 15:41:22,720 DEBUG State transition from <GetObjectParametersState> to <wait_get_obj_params>`
+    1. `2022-04-25 15:41:22,720 DEBUG Sending TR069 message: GetParameterValues {...}`
+    1. `2022-04-25 15:41:22,723 DEBUG 10.0.2.243 - "POST / HTTP/1.1" 200 877`
+    1. `2022-04-25 15:41:22,769 DEBUG Handling TR069 message: GetParameterValuesResponse {...}`
+    1. `2022-04-25 15:41:22,770 DEBUG Received object parameters: {...}`
+
```

Comment:
There is inconsistency in the list item spacing here.

Reply:
Reverted the logs output to raw `console` blocks. I have added a TAB spacing to those blocks to make them look like they are a subsection of the ordered item.